lib/main.py

```diff
@@ -62,9 +64,19 @@ def write_cfg(*args, **kwargs):
 async def hello_world(context):
     channel = context.channel
     author = context.author
-    print(f"Message sent in {channel}")
+    print(f"Message sent in {channel} from {author.id}")
     await channel.send(f"Hello {author}!")
 
+@bot.command(name = 'describe')
+async def describe_item(context):
+    channel = context.channel
+    itemName = context.message.content.replace("-describe ", "")

```

Comment:
Would it not be better to simply pass a second variable after `context` as `item` in the function definition and use that, a-la the `SetRole` command, instead of stripping the command from the message content to get the item name?

Reply:
Yep. I figured there must be a better way to do this.